src/templates/app/user/authenticate.jsx

```diff
@@ -313,17 +358,25 @@ const Authenticate = () => (
                                     <img className='gr-padding-20' src={it.url_for('images/pages/authenticate/valid.svg')} />
                                     <h1>{it.L('Your proof of address has been verified successfully')}</h1>
                                     <p id='text_verified_poi_required' className='invisible'>{it.L('You must also submit a proof of identity.')}</p>
+                                    <p id='text_verified_edd_required' className='invisible'>{it.L('You must also submit a proof of income.')}</p>
```

Comment:
same here, check please do we really  need  income to be connected to other features'